import-extension/build.gradle.kts

```
@@ -43,8 +43,16 @@ publishing {
                 name.set(pomName)
                 packaging = pomPackaging
                 description.set(pomDescription)
+                url.set("https://github.com/pkware/detektExtensions")
                 setPkwareOrganization()
 
+                developers {
+                    developer {
```

This is fine as we clearly didn't care to begin with, but this does allow for identifying the individual contributors using something like

```suggestion
                    developer {
                        id.set("mbayerPK")
                        name.set("Matthew Bayer")
                        email.set("matthew.bayer@pkware.com")
                    }
                    developer {
                        id.set("MariusVolkhart")
                        name.set("Marius Volkhart")
                        email.set("marius.volkhart@pkware.com")
                    }
```

---

PKWARE, Inc., not individuals

---

i amended the pr commit for this please let me know if this is suitable